Gemfile

```diff
@@ -27,6 +27,9 @@ gem "view_component"
 # GOV.UK Notify
 gem "mail-notify"
 
+# Hotwire's modest JavaScript framework [https://stimulus.hotwired.dev]
+gem "stimulus-rails"
+
```

Comment:
I'm new to Stimulus. I've never needed to use it before, but I can see that it brings some advantages. I like the way it defines an explicit relationship between HTML and associated JS behaviour.

However, if this is something that's worth using, then I think we should adopt it wholesale. Thankfully we're still early on so that's not too disruptive of a change to make.

I'd consider this to be an architectural decision, and therefore I think we should probably open a standalone PR which:

- Adds an ADR to explain how/when/why we use Stimulus
- Migrates existing JavaScript within the app to use Stimulus

What do you think @CatalinVoineag?

Reply:
Sure, I'll get that PR merged first

Reply:
Resolved by #152